{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"networkCallFailed"},{"kind":"text","text":"("},{"kind":"externalParam","text":"details"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/fonetworking\/networkrequestererror\/networkcallfailed(details:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/FONetworking\/documentation\/FONetworking\/NetworkRequesterError\/networkCallFailed(details:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"The network call failed to yield a response"}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"networkCallFailed"},{"kind":"text","text":"("},{"kind":"externalParam","text":"details"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"title":"NetworkRequesterError.networkCallFailed(details:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:12FONetworking21NetworkRequesterErrorO17networkCallFailedyACSS_tcACmF","modules":[{"name":"FONetworking"}]},"hierarchy":{"paths":[["doc:\/\/FONetworking\/documentation\/FONetworking","doc:\/\/FONetworking\/documentation\/FONetworking\/NetworkRequesterError"]]},"references":{"doc://FONetworking/documentation/FONetworking/NetworkRequesterError":{"role":"symbol","title":"NetworkRequesterError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NetworkRequesterError"}],"abstract":[{"type":"text","text":"Any error that occured when sending a network request"}],"identifier":"doc:\/\/FONetworking\/documentation\/FONetworking\/NetworkRequesterError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"NetworkRequesterError"}],"url":"\/documentation\/fonetworking\/networkrequestererror"},"doc://FONetworking/documentation/FONetworking/NetworkRequesterError/networkCallFailed(details:)":{"role":"symbol","title":"NetworkRequesterError.networkCallFailed(details:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"networkCallFailed"},{"kind":"text","text":"("},{"kind":"externalParam","text":"details"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"The network call failed to yield a response"}],"identifier":"doc:\/\/FONetworking\/documentation\/FONetworking\/NetworkRequesterError\/networkCallFailed(details:)","kind":"symbol","type":"topic","url":"\/documentation\/fonetworking\/networkrequestererror\/networkcallfailed(details:)"},"doc://FONetworking/documentation/FONetworking":{"role":"collection","title":"FONetworking","abstract":[],"identifier":"doc:\/\/FONetworking\/documentation\/FONetworking","kind":"symbol","type":"topic","url":"\/documentation\/fonetworking"}}}